doc/modules/neighbors.rst

```diff
@@ -834,13 +844,18 @@ Here the ``transform`` operation returns :math:`LX^T`, therefore its time
 complexity equals ``n_components * n_features * n_samples_test``. There is no
 added space complexity in the operation.
 
+|details-start|
 
 .. topic:: References:
 
+|details-split|
+
```

Comment:
This dropdown contains a reference from it's exterior. The other ones can stay but this one we'll have to revert it please.